rust-tests/cbmc-reg/Assume/main.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+fn __VERIFIER_assume(cond: bool) {
+    unimplemented!()
+}
+
+fn __nondet<T>() -> T {
+    unimplemented!()
+}
+
```

Comment:
Should these be hoisted into a general header for RMC? So we don't have to repeat their defs each time?